guides/common/modules/proc_disabling-puppet.adoc

```diff
@@ -0,0 +1,26 @@
+[id="Disabling_Puppet_Integration_{context}"]
+= Disabling Puppet Integration with {Project}
+
+To discontinue using Puppet in your {Project}, follow this procedure.
+
+.Prerequisites
+* You upgraded {Project} from the previous version with Puppet server enabled.
+
+.Procedure
+. Disable Puppet server on {SmartProxies} before you disable it on {ProjectServer}:
```

Comment:
Maybe we should be specific here, someone might realize there is also an internal Smart proxy and might want to try disabling it on that one :)
```suggestion
. Disable Puppet server on external {SmartProxies} before you disable it on {ProjectServer}:
```

Reply:
Changed, thx